manager/notifier.go

```diff
@@ -453,6 +457,49 @@ func (n *notifier) sendWebhookNotification(op notificationOp, config *pb.Webhook
 	return nil
 }
 
+// Amazon SNS
```

Comment:
To be consistent, let's omit this short doc string since the other notification methods don't have one either and the method name is pretty clear.

Any other comments should be sentence-style please (start capitalized, end with period).